probe/core/src/main/java/org/jboss/weld/probe/Reports.java

```diff
@@ -0,0 +1,373 @@
+                            } else if (Default.class.equals(qualifier.annotationType())) {
+                                qualifiers.append(JsonObjects.simplifiedAnnotation(qualifier));
+                            } else {
+                                qualifiers.append(HtmlTag.div().attr(HtmlTag.TITLE, qualifier.toString()).add(abbreviateAnnotation(qualifier.toString())));
+                            }
+                            if (iterator.hasNext()) {
+                                qualifiers.append(HtmlTag.BR);
+                            }
+                        }
+                    }
+                    String beanClass = Formats.formatType(bean.getBeanClass(), false);
+                    table.addAndGet(HtmlTag.tr()).add(HtmlTag.td(++idx + "."), HtmlTag.td().add(HtmlTag.a("#" + BDAS).add(" " + (i + 1))),
+                            HtmlTag.td(BeanKind.from(bean).toString()),
+                            HtmlTag.td().add(HtmlTag.div().attr(HtmlTag.TITLE, beanClass).add(abbreviateType(beanClass))), HtmlTag.td(types.toString()),
+                            HtmlTag.td(qualifiers.toString()), HtmlTag.td(JsonObjects.simplifiedScope(bean.getScope())));
+                }
+            }
+        }
+
+    }
+
+    static List<ConfigurationKey> getSortedConfigurationKeys() {
+        List<ConfigurationKey> configurationKeys = new ArrayList<>();
+        Collections.addAll(configurationKeys, ConfigurationKey.values());
+        Collections.sort(configurationKeys, new Comparator<ConfigurationKey>() {
+            @Override
+            public int compare(ConfigurationKey o1, ConfigurationKey o2) {
+                return o1.get().compareTo(o2.get());
+            }
+        });
+        return configurationKeys;
+    }
+
+    static Object getValue(ConfigurationKey key, WeldConfiguration configuration) {
+        Object defaultValue = key.getDefaultValue();
+        Object value = null;
+        if (defaultValue instanceof Boolean) {
+            value = configuration.getBooleanProperty(key);
+        } else if (defaultValue instanceof Long) {
+            value = configuration.getLongProperty(key);
+        } else if (defaultValue instanceof Integer) {
+            value = configuration.getIntegerProperty(key);
+        } else if (defaultValue instanceof String) {
+            value = configuration.getStringProperty(key);
+        }
+        return value;
+    }
+
+    static String getDesc(ConfigurationKey key) {
+        try {
+            Field field = ConfigurationKey.class.getDeclaredField(key.toString());
+            if (field != null && field.isEnumConstant()) {
+                Description description = field.getAnnotation(Description.class);
+                if (description == null) {
+                    // Don't show config options without description
+                    return null;
+                }
+                return description.value();
+            }
+        } catch (Exception e) {
```

Comment:
FindBugs finds this line not to it's liking it seems:
`Exception is caught when Exception is not thrown in org.jboss.weld.probe.Reports.getDesc(ConfigurationKey) ["org.jboss.weld.probe.Reports"] At Reports.java:[lines 78-370]`